build.sh

```diff
@@ -292,6 +297,7 @@ echo "Get params:
     BUILD_JAVA_EXT              -- $BUILD_JAVA_EXT
     OUTPUT_COMPILE_TIME         -- $OUTPUT_COMPILE_TIME
     WITH_TENANN                 -- $WITH_TENANN
+    WITH_RELATIVE_SRC_PATH          -- $WITH_RELATIVE_SRC_PATH
```

Comment:
```suggestion
    WITH_RELATIVE_SRC_PATH       -- $WITH_RELATIVE_SRC_PATH
```

make it aligned with the previous line.

Reply:
Done.